src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+  
+ - They can't be cloned
+ - They take longer to rot (2x? 3x?)
+ - They have 50% poison resistance (oxygen bypasses this. poison is generally not a very common damage type/not a very useful resistance to have, so I feel 50 is fine)
+ 
+ - They have their own blood type, and when that system is expanded, it should be incompatible with "normal blood"
+ - Possibly their brain, (it actually being a piece of tech), is automatically ready to be placed into a borg chassis
+
+  - They are small/unassuming/fragile.
+    - Possibly a dwarf-sized sprite? (when you sprite it)
+    - They take ~ +25% damage from all Bruise
+    - They get drunk more easily
+    - Maybe they are worse at pulling things?
+    - They are easier to shove
+    - I thought they would deal sub-par damage but there have been some fairly strong feelings on Discord that their claws should be vicious claws. Maybe an exception can be made here, if they take more brute damage they won't really have an advantage in a brawl anyway. Unarmed damage probably isn't really all that important in a real fight anyway)
+  
+- They should have some actual mechanic involving screams (any such ability should consider that the hotbar cooldown does not apply to emoted screams. They might need to be an actual ability that also does a scream, rather than apply on any scream)
+  - they could have the ability to discern the positions of screams within a range, even through walls
+  - they could discern some information from screams, such as the coarse Health state of the person? (Good, Okay, Poor, Bad, Danger)
+  - their screams could apply a ! popup on nearby people with no additional effect?
+
+- They have a weird dietary range.
+  - It has been suggested that they be able to butcher without any tools. Very messily.
+  - Their thirst can only be satiated with welding fuel?
+  - Water makes them drunk
+  - Maybe "normal food" is less satiating to them than the weird stuff (if that can be categorized in some way)
+  
+- This was just an idea I toyed around with but they could have a mechanic based on hugs. However we want to explain it, from a deep psychological need based on their pack hunter ancestry, to them actually sapping small amounts of bioelectrical energy from others with surface contact (and reducing the other person's life expectancy by 1 to 3 seconds. Don't worry you'll be fiiiine it hardly matters.). The point is that they can't hug themselves and this will make them either hug every single crewmember they see (in hopes of reciprocation) and become known as having no concept of personal space, or threaten you with a fireaxe for hugs. Both of these outcomes are weird which is good.
+  - They get a sad vox face debuff on the alert HUD after going X minutes without getting a hug. What a terrible fate 
+  - Their bruise regeneration (that all organics have at low damage levels) could stop?
+  - Maybe their hunger meter depletes faster when debuffed
```

Comment:
ALLOY VULTURES?

Depressed vox sounds funny, but maybe not sapping life energy. Just sounds like an easy way to cause issues, lol. I think losing nutrition faster would be funny, as well as making them slower?

Reply:
Having hugs provide some kind of buff is a really neat idea and aligns with the whole "vox together strong" community identity. I really like this concept but I think it could use a bit more development/certainty. And as I mentioned previously this isn't really a design so much as a rough concept which could use some more iteration.